app/views/pages/instances/create.liquid

```diff
@@ -0,0 +1,17 @@
+---
+method: post
+layout: instances
+slug: instances
+---
+
+{% liquid
+  include 'require_auth'
+
+  function object = 'lib/commands/instances/create', domain: context.params.domain, pos_cli_token: context.params.pos_cli_token, available: true 
+
+  if object.valid
+    redirect_to '/' 
+  else
+    theme_render_rc 'layout/new', errors: object.errors, domain: context.params.domain, pos_cli_token: context.params.pos_cli_token
```

Comment:
just use `render` instead of theme_render_rc and the file path should be `instances/new` I believe :) you should also provide just object instead of all parameters individually